docs/ldml/tr35-keyboards.md

```diff
@@ -44,46 +44,47 @@ The LDML specification is divided into the following parts:
 
 ## <a name="Contents" href="#Contents">Contents of Part 7, Keyboards</a>
 
-*   1 [Keyboards](#Introduction)
-*   2 [Goals and Non-goals](#Goals_and_Nongoals)
-*   3 [Definitions](#Definitions)
-*   4 [File and Directory Structure](#File_and_Dir_Structure)
-*   5 [Element Hierarchy - Layout File](#Element_Heirarchy_Layout_File)
-    *   5.1 [Element: keyboard](#Element_Keyboard)
-    *   5.2 [Element: version](#Element_version)
-    *   5.3 [Element: generation](#Element_generation)
-    *   5.4 [Element: info](#Element_info)
-    *   5.5 [Element: names](#Element_names)
-    *   5.6 [Element: name](#Element_name)
-    *   5.7 [Element: settings](#Element_settings)
-    *   5.8 [Element: keyMap](#Element_keyMap)
-        *   Table: [Possible Modifier Keys](#Possible_Modifier_Keys)
-    *   5.9 [Element: map](#Element_map)
-        *   5.9.1 [Element: flicks, flick](#Element_flicks)
-    *   5.10 [Element: import](#Element_import)
-    *   5.11 [Element: displayMap](#Element_displayMap)
-    *   5.12 [Element: display](#Element_display)
-    *   5.13 [Element: layer](#Element_layer)
-    *   5.14 [Element: row](#Element_row)
-    *   5.15 [Element: switch](#Element_switch)
-    *   5.16 [Element: vkeys](#Element_vkeys)
-    *   5.17 [Element: vkey](#Element_vkey)
-    *   5.18 [Element: transforms](#Element_transforms)
-    *   5.19 [Element: transform](#Element_transform)
-    *   5.20 [Element: reorders, reorder](#Element_reorder)
-    *   5.21 [Element: transform final](#Element_final)
-    *   5.22 [Element: backspaces](#Element_backspaces)
-    *   5.23 [Element: backspace](#Element_backspace)
-*   6 [Element Hierarchy - Platform File](#Element_Heirarchy_Platform_File)
-    *   6.1 [Element: platform](#Element_platform)
-    *   6.2 [Element: hardwareMap](#Element_hardwareMap)
-    *   6.3 [Element: map](#Element_hardwareMap_map)
-*   7 [Invariants](#Invariants)
-*   8 [Data Sources](#Data_Sources)
-    *   Table: [Key Map Data Sources](#Key_Map_Data_Sources)
-*   9 [Keyboard IDs](#Keyboard_IDs)
-    *   9.1 [Principles for Keyboard Ids](#Principles_for_Keyboard_Ids)
-*   10 [Platform Behaviors in Edge Cases](#Platform_Behaviors_in_Edge_Cases)
+* 1 [Keyboards](#Introduction)
+* 2 [Goals and Non-goals](#Goals_and_Nongoals)
+* 3 [Definitions](#Definitions)
+  * 3.1 [Escaping](#Escaping)
+* 4 [File and Directory Structure](#File_and_Dir_Structure)
+* 5 [Element Hierarchy - Layout File](#Element_Heirarchy_Layout_File)
+  * 5.1 [Element: keyboard](#Element_Keyboard)
+  * 5.2 [Element: version](#Element_version)
+  * 5.3 ~~[Element: generation](#Element_generation)~~

```

Comment:
What is the ~~ in this entry?

Reply:
It's strikeout, and is actually in the original element because it is deprecated.  It works in the ToC and so might be a positive change.

Reply:
it's deprecated